noWord/common/DefaultTransformation.py

```diff
@@ -93,6 +93,9 @@ def sort( input, params, context):
     sortKey = params['key']
     reverseFlag = params['reverse'] if 'reverse' in params else False
 
+    if inputRes is None:

```

Comment:
An empty inputRes is often due to a missing file or a misspelled resource name. Thus, being permissive will allow document generation without letting the user know something is wrong.

Is there an use case where inputRes can be undefined ?

Reply:
Yes. I had another transformation before using the selectFirst. When it do not finds anything, it broke. The underlying data is generated dynamically and might not always contains values for all possible keys.